Includes/survival-games.xml

```diff
@@ -55,35 +55,70 @@
 <objective>The rules of the Survival Games are simple. In punishment for the uprising, tributes will be imprisoned in a vast outdoor arena that could hold anything. They must fight to the death. The last tribute standing wins.</objective>
 <constants fallback="true"> <!-- Default values - Can be overriden by an individual map's code -->
     <constant id="timelimit">13m</constant> <!-- Match ending time -->
-    <constant id="deathmatch-pre-teleport">10m45s</constant> <!-- How long until players are warned for teleport to deathmatch -->
-    <constant id="deathmatch-teleport">11m</constant> <!-- How long until players are teleported to the deathmatch -->
-    <constant id="deathmatch-start">11m15s</constant> <!-- How long until the deathmatch starts (after teleporting) -->
-    <constant id="player-count">24</constant> <!-- How many players this map is suited for based on podium counts. Common values: "8", "12", "24", "32", "48" -->
+    <constant id="min-player-count">8</constant> <!-- Minimum player count to start the match -->
+    <constant id="max-player-count">24</constant> <!-- Maximum player count for the match -->
+    <constant id="timelock">on</constant> <!-- Whether to lock world time -->
+    <constant id="difficulty">normal</constant> <!-- Difficulty of the match (peaceful, easy, normal, hard) -->
```

Comment:
I would make sure to mention that difficulty only affects how fast you can regenerate